app/forms/fields/month_year_date_field.py

```diff
@@ -27,11 +29,28 @@ def data(self):
 
 
 class MonthYearDateField(FormField):
-    def __init__(self, validators, **kwargs):
+    def __init__(
+        self,
+        *,
+        validators: Sequence[Callable],
+        label: str | None = None,
+        description: str,
+        **kwargs: Any,
+    ) -> None:
         form_class = get_form_class(validators)
-        super().__init__(form_class, **kwargs)
-
-    def process(self, formdata, data=unset_value, extra_filters=None):
+        super().__init__(
+            form_class,
+            label=label,
+            description=description,
+            **kwargs,
+        )
+
+    def process(
+        self,
+        formdata: MultiDict,
+        data: UnsetValue = unset_value,
```

Comment:
data can be str or unsetvalue. For some reason mypy isn't picking up some cases.